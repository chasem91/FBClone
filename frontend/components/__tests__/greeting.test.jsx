import React from 'react'
import Greeting from '../greeting/greeting'
import renderer from 'react-test-renderer'

test('Greeting displays properly', () => {
  const component = renderer.create(
    <Greeting></Greeting>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
