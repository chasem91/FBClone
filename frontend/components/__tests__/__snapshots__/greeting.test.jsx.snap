exports[`test Greeting displays properly 1`] = `<div />`;
